source/_integrations/vera.markdown

```diff
@@ -31,43 +31,26 @@ There is currently support for the following device types within Home Assistant:
 and will be automatically added when HA connects to your Vera controller.
 
 ## Configuration
+<div class='note info'>
 
-To use Vera devices in your installation, add the following to your configuration.yaml file using the IP and port number of your Vera controller:
+The Vera integration is configured exclusively through the front-end. Manual setup through `configuration.yaml` is deprecated.
```

Comment:
Let not make big warnings. Those are for the release notes and changelogs that go with the releases.

Instead, we should write the documentation for a new user getting here. What does he has to click and do to make it work?

Please write the documentation from a new user perspective.